Exercise 1   In the BRFSS (see Section 5.4), the distribution of heights is roughly normal with parameters µ = 178 cm and σ = 7.7 cm for men, and µ = 163 cm and σ = 7.3 cm for women.
In order to join Blue Man Group, you have to be male between 5’10” and 6’1” (see http://bluemancasting.com). What percentage of the U.S. male population is in this range? Hint: use scipy.stats.norm.cdf.

In [6]:
import scipy.stats
import thinkstats2
import numpy as np

In [4]:
cd '/Users/JennyVyas/Documents/Statistics/ThinkStats2/code'

/Users/JennyVyas/Documents/Statistics/ThinkStats2/code


For men, mean of the distribution is 178cm and standard deviation is 7.7cm.
Percentage of male population between two height values can be calculated by the difference of cdf values at the two values.  6'1" is 185.42 in cm and 5'9" is 175.26 in cm.

In [7]:
scipy.stats.norm.cdf(185.42,178,7.7)-scipy.stats.norm.cdf(175.26,178,7.7)

0.47140722599965384

So 47.14% of U.S. male population is in this range assuming the heights of U.S. men follow a normal distribution.